students/Lincoln_Zhang/Session2/series.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+
+
+num_1 = int(input("Please input the 1st number: "))
+num_2 = int(input("Please input the 2nd number: "))
+
+def fibonacci(n):
+
+	l = [num_1,num_2]
```

Comment:
would be a bit better (and the assignment :-) ) if num_1 and num_2 were passed in to the function.

But you've got the logic down.